src/Stryker.Core/Stryker.Core/StrykerRunner.cs

```diff
@@ -162,6 +162,11 @@ private void AnalyseCoverage(StrykerOptions options)
         /// <returns>The root folder component</returns>
         private IProjectComponent AddRootFolderIfMultiProject(IEnumerable<IProjectComponent> projectComponents, StrykerOptions options)
         {
+            if (!projectComponents.Any())
```

Comment:
Couldn't this also happen if there aren't any files?

Reply:
I mean technically yes. But then there'd be an empty solution or only a test project. Both make no sense. The "no test project" case seems the only reasonable one.

Reply:
We could fail to resolve any files with buildalyzer, and then end up with no project components. The error then send you on the wrong path.

Reply:
I've tried really hard to get separate errors for when there are no test projects vs when there are no projects at all. But I can't get it to work without a huge refactor. 

Basically the issue is that we now skip the initialisation completely if we don't find a test project. So there is no MutationTestProcess created. The same happens when no project is found, because of course we don't create any MutationTestProcess then. But if we want to create a MutationTestProcess for a project without a test project we need to adapt the whole initialisation process to that. I've tried but it's too much work. We also can't throw the exception when we notice a project without any test projects. Because there might be another project that DOES have test project. 

tldr; I've tried but no luck. I say: we keep it like this